docs/coordinates.rst

```diff
@@ -0,0 +1,302 @@
+.. _coordinates::
+
+=============================
+Coordinates and ndcube Objects
+==============================
+In the :ref:`ndcube` section we showed how `~ndcube.NDCube`'s slicing ensures the coordinate transformations remain consistent with the data as it is sliced.  In this section we will discuss the many other ways in which the ndcube classes support the integration of data and its coordinates.
+
+But first let's recreate the data and WCS components of an `~ndcube.NDCube` for use in our demonstration.
+
+.. code-block:: python
+
+  >>> import numpy as np
+  >>> import astropy.wcs
+  >>> data = np.ones((3, 4, 5))
+  >>> wcs_input_dict = {
+  ... 'CTYPE1': 'WAVE    ', 'CUNIT1': 'Angstrom', 'CDELT1': 0.2, 'CRPIX1': 0, 'CRVAL1': 10, 'NAXIS1': 5,
+  ... 'CTYPE2': 'HPLT-TAN', 'CUNIT2': 'deg', 'CDELT2': 0.5, 'CRPIX2': 2, 'CRVAL2': 0.5, 'NAXIS2': 4,
+  ... 'CTYPE3': 'HPLN-TAN', 'CUNIT3': 'deg', 'CDELT3': 0.4, 'CRPIX3': 2, 'CRVAL3': 1, 'NAXIS3': 3}
+  >>> input_wcs = astropy.wcs.WCS(wcs_input_dict)
+
+.. _extra_coords::
+
+ExtraCoords
+===========
+In the :ref:`ndcube` section we saw that the WCS object stored at `nducbe.NDCube.wcs` contains the primary set of coordinate transformations that describe the data.  However, what if you have alternative or additional coordinates that are not represented by the WCS?  The `ndcube.ExtraCoords` class provides users with a mechanism of attaching such coordinates to their `~ndcube.NDCube` instances.  Let's start by creating a `~astropy.time.Time` object representing the times corresponding to each position along the 1st axis of the data array defined above.
```

Comment:
I'd give a list of examples here of what the alternative or additional coordinates are